.github/workflows/composite/sentry-create-and-upload-artifacts/action.yml

```diff
@@ -0,0 +1,49 @@
+---
+name: sentry-create-upload
+description: Create and upload sentry release artifacts
+
+inputs:
+  EXECUTABLE_NAME:
+    required: true
+    default: sessiond
+  SENTRY_ORG:
+    required: true
+    default: lf-9c
+  PATH_TO_EXEC:
+    required: true
+    default: lte/gateway/magma-packages
+
+runs:
+  using: composite
+  steps:
+    - name: Create debug artifacts to be uploaded to Sentry.io
+      run: |
+        # The assumption here is that packages.tar.gz was copied out and untarred
+        # in $MAGMA_ROOT/circleci in the magma_integ_test step
+        cd "${{ inputs.PATH_TO_EXEC }}"
+
+        objcopy --only-keep-debug "${{ inputs.EXECUTABLE_NAME }}" "{{ inputs.EXECUTABLE_NAME }}".debug
+        objcopy --strip-debug --strip-unneeded "{{ inputs.EXECUTABLE_NAME }}"
+        objcopy --add-gnu-debuglink="{{ inputs.EXECUTABLE_NAME }}".debug "{{ inputs.EXECUTABLE_NAME }}"
+    - name: Upload debug artifacts for an executable to Sentry.io
+      env:
+        executable_name: sessiond
+        projects: ["lab-agws-native", "radtonics_lab_native", "magma-staging-native", "si-agw-native"]
```

Comment:
As a note for maybe future improvements, sentry does support just uploading to a custom repository (https://docs.sentry.io/platforms/native/data-management/debug-files/symbol-servers/) . Then maybe we don't have to upload to every new project we add, we can just add it to one place. Maybe I'll add an issue to file this :D